### PR TITLE
db: Move analysis tables to their own schema

### PR DESCRIPTION
**Pull request by mitjat**
Internal refactor. This PR moves analysis DB tables (= those with analyzer-internal info) into their own `analysis` schema, introduced in #448.

Testing: Ran for 2000 blocks with all analyzers; no errors were produced.

### CODE REVIEW

**Review by pro-wh — APPROVED** [review]
this one looks fine. haven't looked at the base branch tho

File: storage/migrations/01_consensus.up.sql [hunk 1]
```diff
@@ -285,7 +285,7 @@ CREATE TABLE chain.accounts_related_transactions
 CREATE INDEX ix_accounts_related_transactions_address_block_index ON chain.accounts_related_transactions (account_address);
 
 -- Indexing Progress Management
-CREATE TABLE chain.processed_blocks
+CREATE TABLE chain.processed_blocks  -- Moved to analysis.processed_blocks by later migrations.
```
Comment (ptrus) [inline storage/migrations/01_consensus.up.sql:288]:
```suggestion
CREATE TABLE chain.processed_blocks  -- Moved to analysis.processed_blocks in 06_analysis_schema.up.sql.
```

**Review by ptrus — APPROVED** [review]

File: storage/migrations/01_consensus.up.sql (hunk 1 above)
Comment (mitjat) [reply storage/migrations/01_consensus.up.sql]: :+1: fixed here and elsewhere

File: storage/migrations/06_analysis_schema.up.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+-- Moves analysis tables (= those that keep track of analyzers' internal state/progess) to the new `analysis` schema.
+
+ALTER TABLE chain.processed_blocks SET SCHEMA analysis;
```
[the remark this answers is not in the document]
Comment (pro-wh) [reply storage/migrations/06_analysis_schema.up.sql:3]: (:

**Pull request merged**